components/image-builder/workspace-image-layer/gitpod-layer/debian/gitpod/.bashrc-prepend

```diff
@@ -10,7 +10,7 @@ export PS1='\[\e]0;\u \w\a\]${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u\
 
 # editor config
 if [ -z "$EDITOR" ]; then
-    export EDITOR="gp open -w"
+    export EDITOR="gp open -w || vi"
```

Comment:
I'm not 100% convinced this will work as expected.

E.g. when you run `git commit` in the Terminal, I believe Git will call something like `eval("$EDITOR .git/COMMIT_MSG")` and then wait for that command to return (success indicates "make the commit", failure indicates "abort and don't make the commit").

I think injecting `|| vi` into this mechanism may wreak havoc. Needs to be tested though.

Reply:
I'm just trying out things 😄